web/docs/migrate-from-0-11-to-0-12.md

```diff
@@ -377,6 +377,63 @@ Between Wasp 0.11.X and 0.12.X, the Dockerfile that Wasp generates for you for d
 We suggest that you temporarily move your custom Dockerfile to a different location, then run `wasp start` to generate the new Dockerfile.
 Check out the `.wasp/out/Dockerfile` to see the new Dockerfile and what changes you need to make. You'll probably need to copy some of the changes from the new Dockerfile to your custom one to make your app work with Wasp 0.12.X.
 
+### Migrating declaration names
```

Comment:
Probably would be better if this went above the Dockerfile section above, since Dockerfile section tells user to use `wasp start` but `wasp start` will fail if names of declarations are incorrectly capitalized, so you really want to take care of that first.

Reply:
Ok, I moved it above dockerfile and also tailwind.
I also shortened the examples, it was redundant. And added mention of `main.wasp` to put it into context.